test/product_from_json_test.dart

```diff
@@ -32,4 +32,35 @@ void main() {
         json.decode(encodedJson) as Map<String, dynamic>;
     Product product2 = Product.fromJson(decodedJson);
   });
+
+  test('Load product from JSON - multilingual categories', () async {
+    String encodedJson =
```

Comment:
I've tried to put the json data in an asset file but I have spent too much time on it without success; it would have been great, though.